[docs] update documentation for integration user-authn with Gitlab ver16+ (#6553)
deckhouse/deckhouse: modules/150-user-authn/docs/USAGE_RU.md

Для этого необходимо:
* **self-hosted**: перейти в `Admin area` -> `Application` -> `New application` и в качестве `Redirect URI (Callback url)` указать адрес `https://dex.<modules.publicDomainTemplate>/callback`, scopes выбрать: `read_user`, `openid`;

> * **cloud gitlab.com**: под главной учетной записью проекта перейти в `User Settings` -> `Application` -> `New application` и в качестве `Redirect URI (Callback url)` указать адрес `https://dex.<modules.publicDomainTemplate>/callback`, scopes выбрать: `read_user`, `openid`.

Reviewer: ```suggestion
* **cloud gitlab.com**: под главной учетной записью проекта перейти в `User Settings` -> `Application` -> `New application` и в качестве `Redirect URI (Callback url)` указать адрес `https://dex.<modules.publicDomainTemplate>/callback`, scopes выбрать: `read_user`, `openid`;
```

> * При использовании GitLab версии 16 и выше, при создании приложения также требуется включить опцию `Trusted`/`Trusted applications are automatically authorized on Gitlab OAuth flow`.

Reviewer: ```suggestion
* (для GitLab версии 16 и выше) включить опцию `Trusted`/`Trusted applications are automatically authorized on Gitlab OAuth flow` при создании приложения.
```

> * При использовании Gitlab версии 16 и выше - при создании приложения также требуется включить опцию `Trusted`/`Trusted applications are automatically authorized on Gitlab OAuth flow`.

Reviewer: ```suggestion
* При использовании GitLab версии 16 и выше, при создании приложения также требуется включить опцию `Trusted`/`Trusted applications are automatically authorized on Gitlab OAuth flow`.
```
